demo/App.svelte

```diff
@@ -31,7 +33,10 @@
   let colorItem = colorItems[1].value;
 </script>
 
+<a href="https://googles.com">say my name</a>
```

Comment:
Not sure if you meant Google, but this website seems cool as well 😆

Reply:
Nope, I deliberately wanted something that my browser didn't visit before)